e2e/self_node_remediation_test.go

```diff
@@ -168,41 +155,6 @@ var _ = Describe("Self Node Remediation E2E", func() {
 			})
 		})
 
-		Context("Unhealthy node (with SNR)", func() {
```

Comment:
Isn't this a valid test case for resource deletion strategy? :thinking:
You need to adapt the test though, don't check for node deletion/recreation, but resouce deletions?
Same for the test above, old line 163 doesn't make sense anymore (can't comment there)